app/services/meta-tags.ts

```diff
@@ -5,7 +5,7 @@ import Intl from 'ember-intl/services/intl';
 import pathJoin from 'ember-osf-web/utils/path-join';
 import toArray from 'ember-osf-web/utils/to-array';
 
-export type Content = string | number | null | undefined;
+export type Content = object | string | number | null | undefined;
```

Comment:
This was a previous concern as well and part of the reason I had originally mocked it out as a separate service. I have checked to ensure other errors were not occurring, but can also create a separate type if that is best.

Reply:
This was a previous concern as well and part of the reason I had originally mocked it out as a separate service. I have checked to ensure other errors were not occurring, but can also create a separate type if that is best.

Reply:
I've added a separate type for the JSON object to not mess with the other Content types. Since we decided to not create a full service we may want to comment on this to prevent future ambiguity.